04.android/06.solving-problems/15.reinstallation issue when using multiple profiles on Android devices/docs.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Problem with reinstalling AdGuard Ad Blocker on Android devices when using the app on multiple accounts'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+The owners of Android 9 and up tablets or smartphones with additional accounts setted might face the problem with AdGuard Ad Blocker removal. If AdGuard is installed on multiple accounts, when you try to delete it from one account, the app remains in the list of applications, but you can’t do anything with it — uninstall or reinstall. This happens because AdGuard is installed in a different user account on the device. 
```

Comment:
```suggestion
In Android 9 and later, if AdGuard Ad Blocker is installed for more than one user account on your device, you may encounter problems with removing the app. When you delete AdGuard Ad Blocker from one user account, the app will still show up on the list of applications, but you can’t uninstall or reinstall it. It happens because AdGuard is installed for another user account on the device. 
```